src/python_testing/TC_MCORE_FS_1_3.py

```diff
@@ -0,0 +1,181 @@
+        self.is_ci = self.check_pics('PICS_SDK_CI_ONLY')
+        self.print_step(0, "Commissioning DUT to TH, already done")
+        self.step(1)
+        # These steps are not explicitly in step 1, but they help identify the dynamically added endpoint in step 1.
+        root_node_endpoint = 0
+        root_part_list = await self.read_single_attribute_check_success(cluster=Clusters.Descriptor, attribute=Clusters.Descriptor.Attributes.PartsList, endpoint=root_node_endpoint)
+        set_of_endpoints_before_adding_device = set(root_part_list)
+
+        passcode = await self.create_device_for_dut_ecosystem()
+        read_result = await self.TH_server_controller.ReadAttribute(self.device_for_dut_eco_nodeid, [(root_node_endpoint, Clusters.BasicInformation.Attributes.UniqueID)])
+        result = read_result[root_node_endpoint][Clusters.BasicInformation][Clusters.BasicInformation.Attributes.UniqueID]
+        asserts.assert_true(type_matches(result, Clusters.Attribute.ValueDecodeFailure), "We were expecting a value decode failure")
+        asserts.assert_equal(result.Reason.status, Status.UnsupportedAttribute, "Incorrect error returned from reading UniqueID")
+
+        discriminator = 3840
+        device_for_dut_ecosystem_vendor_id = await self.read_single_attribute_check_success(cluster=Clusters.BasicInformation, attribute=Clusters.BasicInformation.Attributes.VendorID, dev_ctrl=self.TH_server_controller, node_id=self.device_for_dut_eco_nodeid, endpoint=root_node_endpoint)
+        device_for_dut_ecosystem_product_id = await self.read_single_attribute_check_success(cluster=Clusters.BasicInformation, attribute=Clusters.BasicInformation.Attributes.ProductID, dev_ctrl=self.TH_server_controller, node_id=self.device_for_dut_eco_nodeid, endpoint=root_node_endpoint)
+        # To prevent need for runtime QR code generation test makes some assumptions
+        # about the app that it valiudates here.
+        asserts.assert_equal(passcode, 20202021, "Test implementation assumption incorrect for passcode")
+        asserts.assert_equal(device_for_dut_ecosystem_vendor_id, 0xFFF1, "Test implementation assumption incorrect for vendor ID")
+        asserts.assert_equal(device_for_dut_ecosystem_product_id, 0x8001, "Test implementation assumption incorrect for product ID")
+
+        self.TH_server_controller.OpenCommissioningWindow(nodeid=self.device_for_dut_eco_nodeid, timeout=900, iteration=1000,
+                                                          discriminator=discriminator, option=1)
+
+        setup_qrcode = "MT:-24J0-Q000KA0648G00"
+        self.wait_for_user_input(
+            prompt_msg=f"Using the DUT vendor's provided interface, commission the device using the following parameters:\n"
+            f"- discriminator: {discriminator}\n"
+            f"- passcode: {passcode}\n"
+            f"- setupPinCode: {setup_qrcode}\n"
+            f"If using FabricSync Admin, you may type:\n"
+            f">>> pairing onnetwork 111 {setup_qrcode}")
```

Comment:
Just to be clear, the node id here isn't the same as `device_for_dut_eco_nodeid` but more of an example of whatever random value fabric sync admin can pick independently, right?

Reply:
I changed this to be `<desired_node_id>`